website/docs/software/plugins/store-forward.md

```diff
@@ -0,0 +1,111 @@
+---
+id: store-forward-plugin
+title: Store and forward plugin
+sidebar_label: Store and forward
+---
+
+## About
+
+This is a work in progress and is not yet available.
```

Comment:
Good place to use `Admonitions` i.e:

:::danger

This is a work in progress and is not yet available.

:::

Reply:
Done, didn't know about admonitions, thanks!